data/prod/c0000000025o.json

```diff
@@ -191,7 +191,7 @@
                 "blockId": "8eb06d3b-4c93-9685-f2d1-cba0a9c49966",
                 "type": "paragraph",
                 "model": {
-                  "text": "It was designed by London-based florist Philippa Craddock, who also created the floral displays for St George's Chapel and St George's Hall using locally sourced foliage, [which were later donated to local hospices](/news/articles/c000000000ro)."
+                  "text": "It was designed by London-based florist Philippa Craddock, who also created the floral displays for St George's Chapel and St George's Hall using locally sourced foliage, [which were later donated to local hospices](/news/articles/c0000000025o)."
```

Comment:
Good spot, but have you considered linking this to `/news/articles/c0000000027o` so it's consistent with the non-prod scenarios and link to a different article?